### PR TITLE
Be/c17 updates

### PR DESCRIPTION
**Pull request by beccaelenzil**
- Update learning goals
- Update voice in README from you to we except where explicit directions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,43 +1,40 @@
...
 ### The Online Book Review Site
 
 ![Site Layout](images/layout.png "Site Layout")
 
-In this exercise you will create the HTML to go along with content for 3 pages of HTML.  The pages of your site will be the pieces of an online book review site.  You will structure the content given in the content folder with semantic html tags providing meaning to the content.
+In this exercise, we will create the HTML to go along with content for 3 pages of HTML. The pages of our site will be the pieces of an online book review site. We will structure the content given in the content folder with semantic HTML tags providing meaning to the content.
 
 Each page must have a:
 -  Doctype, head, and body of an HTML    
 -  A header/title Section
```
Comment (spitsfire) [inline README.md:22]:
```suggestion
-  A heading/title section
```

if we are talking `h1`, then it's a heading, not a header

File: README.md [hunk 2]
```diff
@@ -1,43 +1,40 @@
...
 -  A footer with a copyright notice and your name
 
-__[Under no circumstances should you use inline html styles.](http://stackoverflow.com/questions/2612483/whats-so-bad-about-in-line-css)__
+__[Under no circumstances should we use inline HTML styles.](http://stackoverflow.com/questions/2612483/whats-so-bad-about-in-line-css)__
 
 ### Site Homepage
 
-The homepage of your site is named `index.html`. This page will have:  
+The homepage of the site is named `index.html`. This page will have:  
 -  A header section with the title for your book review site
```
Comment (spitsfire) [inline README.md:31]:
```suggestion
-  A heading section with the title for your book review site
```

File: README.md [hunk 3]
```diff
@@ -1,43 +1,40 @@
...
 -  A section for navigation
 -  A main content section with an unordered list of the top-5 book images (see the images folder)
   - Each image should link to their page in Amazon.com or a book review page.
```
Comment (spitsfire) [inline README.md]:
```suggestion
  - Each image should link to their page on Amazon.com or a book review page.
```

**Review by spitsfire — COMMENTED** [review]
made a few syntax changes, that's it

File: README.md (hunk 1 above)
Comment (beccaelenzil) [reply README.md:22]: I think we're saying a [header](https://www.w3schools.com/tags/tag_header.asp), so should we change it to just header from header/title?

**Pull request merged**